App/Redux/TextileRedux.js

```diff
@@ -55,7 +55,7 @@ export const onboardedSuccess = state => {
 }
 
 export const handleImageAdded = (state, {thread, hash, remotePayloadPath}) => {
-  const items = [{ thread, hash, remotePayloadPath, state: 'pending' }, ...state.images.items]
+  const items = [...state.images.items, { thread, hash, remotePayloadPath, state: 'pending' }]
```

Comment:
figured we should just go with it the way we would and rn deal with it

Reply:
huh? so you want oldest photos at the top? RN should/sounds-like-it-does behave fine either way, so we should just choose whichever we want.

Reply:
no, what i saw was it doesn't matter actually. RN is putting our newest on top your way or my way

Reply:
I don't see how that can be. Really?

Reply:
Oh, this is the wrong file for this test. Sorry I didn't realize that earlier. Like i was saying in our chat, the authoritative source for the list of items is in `IpfsNodeRedux`. The list in here just serves as a pool of data to possibly augment items from the authoritative list with.

Reply:
For the list in here, the order doesn't matter at all.